### ISSUE TITLE
how do you calculate the predicted mask area?

### ISSUE THREAD

**Issue by JoyHuYY1412** [zjhuang22/maskscoring_rcnn#48]
In the calculation of `mask_iou_targets`, https://github.com/zjhuang22/maskscoring_rcnn/blob/c20aa625d8f2e92a8e05c8a8307c2427b6e6015f/maskrcnn_benchmark/modeling/roi_heads/mask_head/loss.py#L182. you use th=0.5 to further get the area of the overlap between your predict mask and ground truth mask.

But the `pre_mask` here has not go through sigmoid, so I wonder why you use 0.5 as a threshold.

**Comment by JoyHuYY1412** [comment]
I think I would use 0 here

**Comment by zjhuang22** [comment]
Yes, you are right, I make a mistake here. Thanks for your correction.
